src/FormField/MultiLine.php

```diff
@@ -932,19 +932,19 @@ private function getDummyExpression($exprField, $model)
      *
      * @return int|mixed|string
      */
-    private function getValueForExpression($exprField, $fieldName, $model)
+    private function getValueForExpression($exprField, $fieldName, Model $m)
     {
         switch ($exprField->type) {
             case 'money':
             case 'integer':
             case 'float':
                 // Value is 0 or the field value.
-                $value = $model[$fieldName] ? $model[$fieldName] : 0;
+                $value = $m->get($fieldName) !== '' ? $m->get($fieldName) : 0;
 
                 break;
             default:
                 // Value is "" or field value enclosed in bracket: "value"
-                $value = $model[$fieldName] ? '"' . $model[$fieldName] . '"' : '""';
+                $value = $m->get($fieldName) !== '' ? '"' . $m->get($fieldName) . '"' : '""';
```

Comment:
what if get() returns `null`? Is that possible? This looks like actual change in code